src/app/api/boosts/claim/route.ts

```diff
@@ -0,0 +1,116 @@
+import { PrismaClient } from '@prisma/client';
+import { NextResponse, type NextRequest } from 'next/server';
+import '@/utils/helper';
+import { Network, Alchemy } from "alchemy-sdk";
+
+const settings = {
+    apiKey: process.env.ALCHEMY_ID,
+    network: Network.BASE_MAINNET,
+};
+
+const alchemy = new Alchemy(settings);
+
+async function ownsNFT(userAddress: string, contract: string) {
+    const response = await alchemy.nft.verifyNftOwnership(userAddress, contract);
+    return response;
+}
+
+async function hasToken(userAddress: string, contract: string) {
+    const contracts = [contract];
+
+    const tokenBalanceRes = await alchemy.core.getTokenBalances(userAddress, contracts);
+    const filterByBalance = tokenBalanceRes.tokenBalances.filter((balance: any) => Number(balance.tokenBalance) > 0);
```

Comment:
no any

Reply:
if tokenBalance is a bigint and if you try to convert to a number, you have a risk of overflow. Couldn't you just do `balance.tokenBalance > 0n`?